src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/DenoisedLocatableCopyRatioCollection.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.collections;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.metadata.SampleLocatableMetadata;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.LinearNonLocatableCopyRatio;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.DenoisedLocatableCopyRatio;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.tools.copynumber.GermlineCNVCaller;
+import org.broadinstitute.hellbender.tools.copynumber.PostprocessGermlineCNVCalls;
+
+import java.io.File;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+/**
+ * Collection of denoised copy ratios generated by {@link GermlineCNVCaller} with their corresponding intervals
+ *
+ * @author Andrey Smirnov &lt;asmirnov@broadinstitute.org&gt;
+ */
+public class DenoisedLocatableCopyRatioCollection extends AbstractSampleLocatableCollection<DenoisedLocatableCopyRatio> {
+
+    enum DenoisedLocatableCopyRatioTableColumn {
+        CONTIG,
+        START,
+        END,
+        DENOISED_COPY_RATIO;
```

Comment:
This should just be `LINEAR_COPY_RATIO`.

Reply:
Done